### ISSUE TITLE
shouldn't use softmax

### ISSUE THREAD

**Issue by hongguangguo** [nfmcclure/tensorflow_cookbook#87]
https://github.com/nfmcclure/tensorflow_cookbook/blob/master/09_Recurrent_Neural_Networks/02_Implementing_RNN_for_Spam_Prediction/02_implementing_rnn.py#L120

shouldn't use softmax  activation in output layer, when use `sparse_softmax_cross_entropy_with_logits`

refer:https://www.tensorflow.org/api_docs/python/tf/nn/sparse_softmax_cross_entropy_with_logits

**Comment by nfmcclure** [comment]
Hi @hongguangguo ,  Thanks, This has been fixed (in two files). Please reopen if you find this occurring anywhere else.
